core/sodasql/cli/cli.py

```diff
@@ -525,6 +525,24 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple, time: st
     default=None,
     type=Path,
 )
+@click.option(
+    "--dbt-cloud-api-token",
```

Comment:
I have it loaded as env var in my shell and reference it like `$MY_SUPER_SECRET_API_KEY`

Reply:
I think I'm missing something then. I don't see this code change parsing any env vars related to the API token of dbt cloud.

I would expect that our env var file parsing now also tries to look for something like `DBT_CLOUD_API_TOKEN`.

I don't think it makes a lot of sense to feed that via the CLI. If anything, because it's inconsistent with how soda-sql gets this kind of thing (cf. soda cloud token etc).

My guess would be that it's better to have it in there and simply remove this option from the CLI.

I think env_vars parsing is happening in this file: https://github.com/sodadata/soda-sql/blob/main/core/sodasql/scan/env_vars.py

But maybe check with @vijaykiran for advice.

Reply:
I don't think we should encourage adding keys as parameters, it would be consistent with other parameters (passwords/tokens storing in the .env_vars). They can always be loaded from shell environment vars by pointing them in the yml file though.

Reply:
In that case: could you specify where exactly the environment variable should go? Are we going to add a new section for this? Or add it to the soda account?

Reply:
Personally, I would indeed expect them to be in the .env_vars file as we do for other passwords and tokens and make the ingest or dbt package read from that env var?

@vijaykiran, is that also in line with your thinking?